registry/download.go

```diff
@@ -55,8 +62,10 @@ func DownloadPluginFromGithub(ctx context.Context, localPath string, org string,
 	}
 	defer archive.Close()
 
-	pathInArchive := fmt.Sprintf("plugins/%s/%s", typ, name)
-	if org != "cloudquery" {
+	var pathInArchive string
+	if org == "cloudquery" && urlIndex == 0 {
```

Comment:
This indirect comparison looks a bit smelly but for now it's ok I think, as other methods would make it more complicated

Reply:
Yeah, agreed... I think I improved it a little bit now by using a struct around the urls instead, PTAL

Reply:
Nice